### ISSUE TITLE
Redirect page preview toolbar not fixed to bottom of viewport

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-cms#1945]
You can't close the preview as you can't get to the actions. 
<img width="577" alt="pasted_image_31_08_17__3_40_pm" src="https://user-images.githubusercontent.com/555033/29905433-d5e51096-8e62-11e7-9ad1-3e58335e7b21.png">

### PRs

- [x] https://github.com/silverstripe/silverstripe-cms/pull/2572 << - merge this first, then rerun tests on admin
- [x] https://github.com/silverstripe/silverstripe-admin/pull/1086

Blocked by https://github.com/silverstripeltd/product-issues/issues/450

**Comment by chillu** [comment]
Removing milestone, it's a fringe use case (previewing redirector pages isn't common I could imagine)

**Comment by robbieaverill** [comment]
Still a bug

**Comment by clarkepaul** [comment]
A few options:
1. Close the preview panel for this page type (disable the preview mode toggle)
2. Have a no no "preview available message - show URL"
3. Show the page you are directing the person too

I think the opt. 1 would probably be the easiest and most practical approach, and opt. 3 is the nicest for the user (might not be easy to do though?).

**Comment by phptek** [comment]
I just came across this yesterday myself in 4.4.3. Doesn't look too polished does it!?

**Comment by clarkepaul** [comment]
Left a message in the PR about it passing my criteria, looks good. 

I've opened another issue to take care of some other related stuff... Contrast of messaging when no preview shows, and centering of message when viewing the message at different device sizes captured here https://github.com/silverstripe/silverstripe-cms/issues/2671

**Comment by maxime-rainville** [comment]
Merged the first PR and rerunning the build on the second PR.

**Comment by maxime-rainville** [comment]
Build on second PR is failing becasue of https://github.com/silverstripeltd/product-issues/issues/450

**Comment by maxime-rainville** [comment]
All merge
